src/main/resources/react/src/redux/profile/saga/workers/UserInfoGet.js

```diff
@@ -3,13 +3,17 @@ import { call, put } from 'redux-saga/effects';
 import { getUserInfo as getUserInfoAsync } from '../../../../REST/Resources';
 import { fillUserInfoSync } from '../../profileAction';
 import { startFetching, stopFetching } from '../../../ui/action';
+import { unauthorized } from '../../../auth/action';
 
-export function* userInfoGet() {
+export function* workerUserInfoGet() {
```

Comment:
The logic is duplicated 4 times into:
UserChildrenDelete.js
UserChildrenPost.js
UserChildrenPut.js
and here.

Why don't you create one file for all those requests?